ARM.CMSIS_DFP.pdsc

```diff
@@ -493,7 +493,7 @@ and 8-bit Java bytecodes in Jazelle state.
           <!-- startup / system file -->
           <file category="sourceC"      name="Device/ARMCM0plus/Source/startup_ARMCM0plus.c"     version="2.0.3" attr="config"/>
           <file category="linkerScript" name="Device/ARMCM0plus/Config/ARMCM0plus_ac6.sct"       version="1.0.0" attr="config" condition="ARMCC6"/>
-          <file category="linkerScript" name="Device/ARMCM0plus/Config/ARMCM0plus_gcc.ld"        version="2.1.0" attr="config" condition="GCC"/>
+          <file category="linkerScript" name="Device/ARMCM0plus/Config/ARMCM0plus_gcc.ld"        version="2.1.1" attr="config" condition="GCC"/>
```

Comment:
```suggestion
          <file category="linkerScript" name="Device/ARMCM0plus/Config/ARMCM0plus_gcc.ld"        version="2.2.0" attr="config" condition="GCC"/>
```

Reply:
Do you suggest to increase the minor version?

Reply:
You are right. versions should be the same as in the files.

Reply:
No, either way. I think you decided to go with a minor version rather than a patch one. In this case both would be fine for me. We just should use consistent versions everywhere.